configs/SUB250REDFOXF722AIO/config.h

```diff
@@ -0,0 +1,126 @@
+#define USE_GYRO_SPI_ICM42688P
+#define USE_ACC_SPI_ICM42688P
+#define USE_BARO_DPS310
+#define USE_BARO_BMP280
+#define USE_FLASH
+#define USE_FLASH_W25Q128FV
+#define USE_MAX7456
+
+#define BEEPER_PIN           PC13
+#define MOTOR1_PIN           PC8
+#define MOTOR2_PIN           PC9
+#define MOTOR3_PIN           PA8
+#define MOTOR4_PIN           PA9
+#define MOTOR5_PIN           PB0
+#define MOTOR6_PIN           PB1
+#define MOTOR7_PIN           PA10
+#define MOTOR8_PIN           PB4
+#define LED_STRIP_PIN        PB3
+#define UART1_TX_PIN         PB6
+#define UART2_TX_PIN         PA2
+#define UART4_TX_PIN         PA0
+#define UART5_TX_PIN         PC12
+#define UART6_TX_PIN         PC6
+#define UART1_RX_PIN         PB7
+#define UART2_RX_PIN         PA3
+#define UART3_RX_PIN         PB11
+#define UART4_RX_PIN         PA1
+#define UART5_RX_PIN         PD2
+#define UART6_RX_PIN         PC7
+#define I2C1_SCL_PIN         PB8
+#define I2C1_SDA_PIN         PB9
+#define LED0_PIN             PC14
+#define LED1_PIN             PC15
+#define SPI1_SCK_PIN         PA5
+#define SPI2_SCK_PIN         PB13
+#define SPI3_SCK_PIN         PC10
+#define SPI1_SDI_PIN         PA6
+#define SPI2_SDI_PIN         PB14
+#define SPI3_SDI_PIN         PC11
+#define SPI1_SDO_PIN         PA7
+#define SPI2_SDO_PIN         PB15
+#define SPI3_SDO_PIN         PB5
+#define ADC_VBAT_PIN         PC1
+#define ADC_CURR_PIN         PC3
+#define PINIO1_PIN           PC0
+#define PINIO2_PIN           PC2
+#define FLASH_CS_PIN         PA15
+#define MAX7456_SPI_CS_PIN   PB12
+#define GYRO_1_EXTI_PIN      PC4
+#define GYRO_2_EXTI_PIN      PB10
+#define GYRO_1_CS_PIN        PA4
+#define GYRO_2_CS_PIN        PB2
+
+#define TIMER_PIN_MAPPING \
+    TIMER_PIN_MAP( 0, PC8 , 2,  0) \
+    TIMER_PIN_MAP( 1, PC9 , 2,  0) \
+    TIMER_PIN_MAP( 2, PA8 , 1,  0) \
+    TIMER_PIN_MAP( 3, PA9 , 1,  0) \
+    TIMER_PIN_MAP( 4, PB0 , 2,  0) \
+    TIMER_PIN_MAP( 5, PB1 , 2,  0) \
+    TIMER_PIN_MAP( 6, PA10, 1,  0) \
+    TIMER_PIN_MAP( 7, PB4 , 1,  0) \
+    TIMER_PIN_MAP( 8, PB3 , 1,  0)
+
+
+#define ADC3_DMA_OPT        0
+
+#define DEFAULT_GYRO_TO_USE GYRO_CONFIG_USE_GYRO_BOTH
+#define MAG_I2C_INSTANCE (I2CDEV_1)
+#define USE_BARO
+#define BARO_I2C_INSTANCE (I2CDEV_1)
+#define USE_ADC
+#define ADC_INSTANCE ADC3
+#define DEFAULT_BLACKBOX_DEVICE     BLACKBOX_DEVICE_FLASH
+#define DEFAULT_DSHOT_BURST DSHOT_DMAR_ON
+#define DEFAULT_CURRENT_METER_SOURCE CURRENT_METER_ADC
+#define DEFAULT_VOLTAGE_METER_SOURCE VOLTAGE_METER_ADC
+#define DEFAULT_CURRENT_METER_SCALE 250
+#define BEEPER_INVERTED
+#define MAX7456_SPI_INSTANCE SPI2
+#define PINIO1_CONFIG 129
+#define PINIO2_CONFIG 129
+#define PINIO1_BOX 40
+#define PINIO2_BOX 41
+#define FLASH_SPI_INSTANCE SPI3
+#define USE_SPI_GYRO
+#define GYRO_1_SPI_INSTANCE SPI1
+#define GYRO_1_ALIGN CW90_DEG
+#define GYRO_1_ALIGN_YAW 900
+#define GYRO_2_SPI_INSTANCE SPI1
+#define GYRO_2_ALIGN CW90_DEG
+#define GYRO_2_ALIGN_YAW 900
+#define align_board_yaw 315
```

Comment:
Why 315 as the unified target uses -45 ???

```suggestion
#define align_board_yaw -45

```